### DISCUSSION TITLE
NOQUEUE: reject: RCPT from unknown[XXX.127.97.227]

### DISCUSSION THREAD

**Discussion by jawwadnaeem** [modoboa/modoboa#3172] [Help] [answered]
I am using modoboa email server i have setup a domain example.com. I have created 2 accounts account1@example.com and account2@example.com. I have setup both accounts in WordPress site in Plugin WP Mail SMTP. I am using one address to send email from out Contact-us form on WP site and receiving emails on second account. But when i submit my information in contact-us form is am getting below error in mail.log

**mailserver postfix/submission/smtpd[3967987]: NOQUEUE: reject: RCPT from unknown[XXX.127.97.227]: 553 5.7.1 <account1@example.com>: Sender address rejected: not owned by user account2@example.com; from=<account1@examle.com> to=<account1@examle.com> proto=ESMTP helo=<mediq.com.pk>**

Any help would be appreciated

**Comment by tonioo** [comment, marked as answer]
You can't send a message from account1@example.com if you're connected with account2@example.com...
